### ISSUE TITLE
Support arbitrary user id

### ISSUE THREAD

**Issue by gulien** [gotenberg/gotenberg#1049]
### Discussed in https://github.com/gotenberg/gotenberg/discussions/1042

<div type='discussions-op-text'>

<sup>Originally posted by **leonidas-o** November 12, 2024</sup>
I see a permission denied error in my open shift/okd cluster:
```
{"level":"error","ts":1731426508.9073021,"logger":"api","msg":"convert HTML to PDF: convert to PDF: process first start: start process: run exec allocator: chrome failed to start:\nmkdir: cannot create directory '//.local': Permission denied\ntouch: cannot touch '//.local/share/applications/mimeapps.list': No such file or directory\nchrome_crashpad_handler: --database is required\nTry 'chrome_crashpad_handler --help' for more information.\n","trace":"f031abb3-c554-41ca-ba8c-745b65b73ba2","remote_ip":"x.x.x.x","host":"gotenberg:3000","uri":"/forms/chromium/convert/html","method":"POST","path":"/forms/chromium/convert/html","referer":"","user_agent":"","status":500,"latency":794288676,"latency_human":"794.288676ms","bytes_in":167432,"bytes_out":21}
```

In the past I had already issues with other images, especially when I see something like: https://gotenberg.dev/docs/getting-started/installation#kubernetes
where it says, I should use `runAsUser: 1001`, which you actually don't do in OpenShift.

This could be resolved with the following in the Dockerfile.
"OpenShift: Support arbitrary user ids"
https://docs.openshift.com/container-platform/4.15/openshift_images/create-images.html#use-uid_create-images
```
RUN chgrp -R 0 /home/gotenberg && \
    chmod -R g=u /home/gotenberg
```

Any one here, who could bring up Gotenberg in a restricted cluster like OpenShift/Okd (maybe without creating a custom Gutenberg image)?</div>

**Comment by leonidas-o** [comment]
@gulien  any news on that?

**Comment by gulien** [comment]
No, I did not find the time. Plus, I got a newborn at home, so I don’t have a lot of bandwidth atm 😄

**Comment by leonidas-o** [comment]
oh no, too bad (just the first part), but congratulations for the second part of your message, your newborn. That's amazing!
